lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1855,7 +1855,8 @@ Value *SPIRVToLLVM::transValueWithoutDecoration(SPIRVValue *BV, Function *F,
     } else if ((OCLSPIRVBuiltinMap::rfind(OC, nullptr) ||
                 isSubgroupAvcINTELInstructionOpCode(OC) ||
                 isIntelSubgroupOpCode(OC)) &&
-               !isAtomicOpCode(OC) && !isGroupOpCode(OC) && !isPipeOpCode(OC)) {
+               !isAtomicOpCode(OC) && !isGroupOpCode(OC) && !isPipeOpCode(OC) &&
+               !isMediaBlockINTELOpcode(OC)) {
```

Comment:
Should we factor it out to some helper function?

Reply:
Sure, I'll try to factor it out later this day.

Reply:
Done

Reply:
Please do the same in #381